spec/javascripts/pages/newReferenceForm.spec.js

```diff
@@ -0,0 +1,80 @@
+describe('newReferencForm', function() {
```

Comment:
noting: there are no tests for user interaction with this widget. is that because the interaction is trivial?